plugins/win-capture/game-capture.c

```diff
@@ -38,10 +42,14 @@
 #define SETTING_CAPTURE_OVERLAYS "capture_overlays"
 #define SETTING_ANTI_CHEAT_HOOK  "anti_cheat_hook"
 #define SETTING_HOOK_RATE        "hook_rate"
+#define SETTING_AUTO_FIT_TO_OUTPUT "auto_fit_to_output"
+#define SETTING_AUTO_LIST_FILE   "auto_capture_list_path"
+#define SETTING_AUTO_WAITING_IMG "auto_waiting_image"
```

Comment:
Having troubles to understand the use/need of this setting.

Reply:
"auto_fit_to_output" as we made auto mode for game capture it will be convenient for a user to have it scaled to stream output by default. And if user manually change size of item, frontend disable this option and manualy set scale will apply. 
"auto_capture_list_path" this setting for a frontend to set a path to a file with list of games. 
"auto_waiting_image" this setting for a frontend to set a path for image what source can show if no game captured.

Reply:
Ok sounds good, makes sense 👍